graphql/schema.graphql

```diff
@@ -59,6 +59,73 @@ type UserCount {
   user: User
 }
 
+type HashtagCount {
+    count: Int
+    name: String
+}
+
+type UserTweet {
+   screen_name: String
+   name: String
+   profile_pic: String
+   createdDatetime: DateTime
+   text: String
+}
+
+extend type Me {
+   topMentions(first: Int = 5): [UserCount] 
+    @cypher(
+      statement: """
+      MATCH (this)-[:POSTS]->(t:Tweet)-[:MENTIONS]->(m:User)
+      WITH m, COUNT(m.screen_name) AS count 
+      ORDER BY count DESC
+      LIMIT $first
+      RETURN {
```

Comment:
I think here you want

```
RETURN {
        user: m {
          .*
        },
        count: count
      }
```

since `user` is a nested object in the `UserCount` type

Reply:
Ok, fixed! Thanks for the feedback.